README.md

```diff
@@ -164,6 +164,12 @@ Rationale: some labels are not very selective, or very rarely come up in queries
 - Mentor(s): kadis (@kadisi), Kevin Wang (@kevin-wangzefeng)
 - Issue： https://github.com/kubeedge/kubeedge/issues/1437
 
+#### KubeEdge AI application
```

Comment:
Can you please add the upstream issue link?

Reply:
Added the issue link. Thanks.